pkg/volume/csi/csi_plugin.go

```diff
@@ -914,3 +919,30 @@ func highestSupportedVersion(versions []string) (*utilversion.Version, error) {
 	}
 	return highestSupportedVersion, nil
 }
+
+// waitForAPIServerForever waits forever to get the APIServer Version as a proxy
```

Comment:
Please modify this comment to be consistent with the node retrieval on line 934

Reply:
fixed